README.md

```diff
@@ -55,6 +55,16 @@ stack exec baseDAO -- document -n BaseDAO
 
 This command will dump the documentation to `BaseDAO.md` file.
 
+## How to write your own DAOs
+
+This repository contains a [template for a DAO contract](/src/Lorentz/Contracts/TemplateDAO.hs) that can be used as an example.
+
+To create your own DAO contract:
+1. Either fork this repository or add a separate Haskell project that depends on this repository;
+2. Create a new Haskell module as a copy of the mentioned template;
```

Comment:
If one forks this repository, I think there is no need to copy the template, one can modify it in place. One _may_ rename this module if they want, but it's really just an aesthetical thing. Also updating `Main.hs` is not really necessary (maybe except for renaming). So I think there are 2 possible flows:
1. You fork this repo and modify code in place. Modifying in place seems more convenient than creating new modules, there are less actions required on your side. One drawback is that you have to manually sync up with upstream changes, you basically maintain a copy of another repo.
2. You create a new project with `baseDAO` directory, in that case you indeed need to make a copy of the template.

Considering what I wrote above, maybe let's make another _package_ in this repo that would depend on `baseDAO` and serve template purpose? Current template is helpful for (1), but is less useful for (2) because in (2) one still needs to manually create a new package. I think this new template package needs just an executable, config can be put there. Probably it's not a good idea to include this template into `baseDAO` library because people who use approach (2) don't want to depend on this module (it's small, but still some extra code).

Reply:
Okay, I think this should be the most convenient thing for users (though potentially pretty inconvenient for us).

Then I remove the last commit and leave a reference to the new project instead.

Reply:
Okay, I think this should be the most convenient thing for users.

Will drop this commit and add a new one instead.